.gitignore

```diff
@@ -25,7 +25,8 @@ webpack-stats.json
 *.sql
 
 # Sensitive environment files
-.env
+**/**/.env
+*.env
```

Comment:
I think this will remove local.env needed for the first steps setup on our docs https://docs.gitcoin.co/mk_setup/